R/tmb.R

```diff
@@ -392,7 +411,7 @@ h_mmrm_tmb_assert_opt <- function(tmb_object,
   }
 }
 
-#' Extract covariance matrix from `TMB` report and input data
+#' Extract covariance matrix from `TMB` report and tmb data
```

Comment:
```suggestion
#' Extract covariance matrix from `TMB` report and input data
```
already says TMB upfront, point is that `tmb_data` is the input data